dedibox/vps/how-to/create-vps.mdx

```diff
@@ -0,0 +1,38 @@
+---
+meta:
+  title: How to create a Dedibox VPS
+  description: This page explains how to create a Dedibox VPS
+content:
+  h1: How to create a Dedibox VPS
+  paragraph: This page explains how to create a Dedibox VPS
+tags: vps dedibox order create
+dates:
+  validation: 2024-05-15
+  posted: 2024-05-15
+categories:
+  - dedibox-vps
+---
+
+A Dedibox VPS (Virtual Private Server) is a private virtualized server that operates within a larger physical infrastructure.
+It provides directly accessible computing resources and greater control of your hosting environment than shared hosting solutions.
+
+VPS are suitable for various use cases, including hosting websites, web applications, databases, development environments, etc. They balance performance, control, and affordability for businesses and individuals with moderate to high-traffic websites or applications.
+
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- [Added a payment method](/dedibox-console/account/how-to/add-payment-method) to your account
+
+1. Open your [Dedibox console](https://console.online.net) home page and click **Order** in the navigation at the top of the page.
+2. From the drop-down menu, select **Dedibox VPS** to access the Dedibox VPS offers page.
+3. Choose the VPS offer you want to deploy, and proceed by clicking **Order**.
+4. Specify the geographical region where you want your VPS deployed.
+5. Select your preferred location for the virtual machine deployment, then click **Order** to proceed.
+6. Choose your virtual machine's desired Operating System (OS) and select the appropriate OS version.
+7. Enter a hostname for your VPS, ensuring it contains at least eight (8) alphanumeric characters. 
+8. Optionally paste an [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) that can be used for authentication when connecting to the Dedibox VPS using SSH, then click Order. An order summary displays.
+9. Review all the parameters for your virtual machine and check the message boxes to indicate you have read the terms of service and specific VPS conditions.
+10. Click **Confirm my order** to confirm your purchase and initiate the deployment of your VPS.
+	<Message type="note">
+        It may take a few minutes for your VPS to be delivered and visible in your account.
+    </Message>
```

Comment:
The message box displays weirdly here, I think there's an issue with the indentation